test/functionalTest/cases/714_propagate_service_path/propagate_service_path_for_update_and_query_standard_ops.test

```diff
@@ -0,0 +1,588 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Propagate Service-Path for Update and Query Requests
+
+--SHELL-INIT--
+dbInit CB
+dbInit CP1
+dbInit ftest-t1
```

Comment:
In this case, we are using only one tenant (the default one, contextBroker is not started with -multiservice)... maybe this is due to copy-paste from the previous .test?